featuretools/primitives/standard/transform/time_series/expanding/expanding_count.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import Datetime, Double
+
+from featuretools.primitives.base.transform_primitive_base import TransformPrimitive
+from featuretools.primitives.standard.transform.time_series.utils import (
+    _apply_gap_for_expanding_primitives,
+)
+
+
+class ExpandingCount(TransformPrimitive):
+    """Computes the expanding count of events over a given window.
+
+    Description:
+        Given a list of datetimes, return an expanding count starting
+        at the row `gap` rows away from the current row. An expanding
+        primitive calculates the value of a primitive for a given time
+        with all the data available up to the corresponding point in time.
+
+        Input datetimes should be monotonic.
+
+    Args:
+        gap (int, optional): Specifies a gap backwards from each instance before the
+            usable data begins. Corresponds to number of rows. Defaults to 1.
+        min_periods (int, optional): Minimum number of observations required for performing calculations
+            over the window. Defaults to 1.
+
+
+    Examples:
+        >>> import pandas as pd
+        >>> expanding_count = ExpandingCount()
+        >>> times = pd.date_range(start='2019-01-01', freq='1min', periods=5)
+        >>> expanding_count(times).tolist()
+        [0.0, 1.0, 2.0, 3.0, 4.0]
```

Comment:
With a default gap of 1, why is the first value here `0.0` instead of `nan`? There shouldn't be any data to count before the first point.

Reply:
It turns out that calling shift on a `pd.date_range` object does not insert `nan` values at the beginning